lib/plottr_components/src/components/dialogs/EditOrDisplay.js

```diff
@@ -117,7 +118,7 @@ const EditOrDisplay = ({
               block
               title={t('No color')}
               bsStyle="warning"
-              onClick={() => setValue('none')}
+              onClick={() => setValue(`${isDarkMode ? 'none' : '#0B1117'}`)}
```

Comment:
Can we leave this as `none` here and instead change how we render `none` later.
Otherwise, it makes things a bit more complicated than they need to be.